app/helpers/application_helper.rb

```diff
@@ -32,4 +32,9 @@ def assocation_empty?(assoc)
     assoc.reload unless assoc.nil? || assoc.loaded?
     assoc.nil? ? true : assoc.size == 0
   end
+
+  def i18n_time_ago_in_words(past_time)
+    "#{time_ago_in_words(past_time)} #{t('time_ago')}"
```

Comment:
If not we can fix it in a separate PR? Or do as is generally better in a separate PR?